addon/data/shoot-panel.js

```diff
@@ -135,6 +141,17 @@ class ShootPanel extends React.Component {
     self.port.emit("copyLink", this.props.shot.viewUrl);
```

Comment:
Ok, I moved the property accesses out of shoot-panel and into shooter. Now the messages will always be consistent with what is copied.